src/base-commands/base-command.js

```diff
@@ -59,13 +65,16 @@ class BaseCommand extends Command {
         propNames.forEach(p => {
           if (fullItem[p] === undefined) {
             invalidPropertyNames.add(p);
+          } else if (fullItem[p] instanceof Date) {
+            const dateString = fullItem[p].toString();
```

Comment:
We might want to use a [`Date`](https://www.w3schools.com/js/js_date_formats.asp) object here so that in `sanatizeProperty` we have finer grained control on the output.

Reply:
That was the original approach, but it was difficult to test. Required mocking timezones which produced non-confirming outputs. I _do_ think that it should now be renamed to `sanitizeDateString` to make it more specific.